docs/getting-started.md

```diff
@@ -15,7 +15,17 @@ The interactive course with quizzes will guide you through everything you need t
 - [Node.js 10](https://nodejs.org/) or later
 - MacOS, Windows (including WSL), and Linux are supported
 
-## Setup
+## Guided Setup
+
+Start a new Next.js app using `create-next-app`:
+
+```bash
+npm init next-app
```

Comment:
TBH I prefer `npx`. Is there any reason we used `npm init`?

Reply:
Is being used by all examples

Reply:
`zeit/docs` [migrated to `npm` from `npx` last year](https://github.com/zeit/docs/pull/1222) and I've updated examples in Next.js accordingly.